### ISSUE TITLE
Custom theme is not applied

### ISSUE THREAD

**Issue by mcodato** [ntop/ntopng#8546]
**Environment**:
* OS name: Red Hat Enterprise Linux
* OS version: 8.10
* Architecture: x86_64
* ntopng version/revision: 
```
Version:	6.0.240703 [Enterprise/Professional build]
GIT rev:	6.0-stable:1e320c2fe082d0a7776993b41aad55d4255aa1cc:20240703
Pro rev:	r6175
```

**What happened**:
The custom theme file is not applied.

**How did you reproduce it?**
1. Create the file `/usr/share/ntopng/httpdocs/css/custom-theme.css`
2. Refresh the ntopng UI with the dev tools opened
3. There are no requests that load the custom theme file 

**Debug Information**:
The problem seems to be here
https://github.com/ntop/ntopng/blob/dev/scripts/lua/modules/page_utils.lua#L377
the custom theme is loaded from `/usr/share/ntopng/httpdocs/css/custom-theme.css` but the `if` checks if the file `/usr/share/ntopng/css/custom-theme.css` exists.
Essentially, the two paths differ in the `httpdocs` folder.

**Comment by MatteoBiscosi** [comment]
Thanks for the fix @mcodato , your pr is correct, in case the issue persists let me know.

**Comment by mcodato** [comment]
Thank you @MatteoBiscosi, will the fix be backported to version 6.0?

**Comment by cardigliano** [comment]
@mcodato a new stable will be released in a few days and will contain the fix
